.buildkite/pipeline.release-experimental.yml

```diff
@@ -40,6 +40,20 @@ steps:
           volumes:
             - "/yum.buildkite.com"
 
+  - name: ":redhat: Publish Edge RPM Package to Buildkite Packages"
+    command: ".buildkite/steps/publish-rpm-buildkite-packages.sh"
+    env:
+      REPOSITORY: "buildkite/agent-rpm-experimental"
+    agents:
+      queue: "deploy"
+    plugins:
+      - docker#v5.8.0:
+          image: "public.ecr.aws/docker/library/ruby:3.0"
```

Comment:
afaict we aren't actually using ruby at all - can we make this `alpine` or `debian:$WHATEVER_DEBIAN_VERSION_PEOPLE_LIKE_USING_THESE_DAYS` or something?

Reply:
Yup, happy to use whatever. I was using the same image so that it might already be warm in the cache, and it has curl installed. Neither alpine nor debian have curl installed.

`032379705303.dkr.ecr.us-east-1.amazonaws.com/deploytools:2022.07` seems to be used by other steps and has curl, I'll swap to that.

Reply:
Worth nothing, what these script are currently doing should be folded into `bk` or a plugin over time, hopefully without needing docker involved at all.

Reply:
cool, looking forward to it!